src/main/resources/XWiki/MapMacro.xml

```diff
@@ -664,18 +680,25 @@ No content.
 = Exemples =
 
 == Simple map ==
-{{code}}
-{{map location="10, rue Pernety, France"/}}
-{{/code}}
+(% class="box"%)((( (% class="code"%)(((
+{{{ {{map location="10, rue Pernety, France"/}} }}}
+))))))
```

Comment:
I want to avoid adding a dependency of the code macro that is used just for the exmple, see this :
https://github.com/xwiki-contrib/macro-map/pull/6